### DISCUSSION TITLE
About React Native

### DISCUSSION THREAD

**Discussion by sonjaie** [learningeconomy/LearnCard#326] [Help] [answered]
Hi Good day, when i tried to implement this line of code

- yarn add @learncard/init
- import { initLearnCard } from '@learncard/init';
- const learnCard = await initLearnCard({ seed: '8f476ba148360c7d1830beb2520e6bf883e934a7d16159ce0930e3cc53799fd9' });

something like my app or metro laggy or Stop? is there something I need to configure?
" when I try to stop the metro and re-run the app, the bundling stop at 99.2% "

**Comment by Custard7** [comment, marked as answer]
> something like my app or metro laggy or Stop? is there something I need to configure? " when I try to stop the metro and re-run the app, the bundling stop at 99.2% "

Hey @sonjaie - thanks for opening a discussion!

I'm not sure I follow your question. Are you able to provide a screenshot of the build issue / provide more detailed steps to recreate, including your system and setup information?
